service_configuration_lib/spark_config.py

```diff
@@ -26,6 +26,8 @@
     'spark.kubernetes.authenticate.clientKeyFile',
     'spark.kubernetes.authenticate.clientCertFile',
     'spark.kubernetes.container.image.pullPolicy',
+    'spark.kubernetes.executor.label.yelp.com/paasta_service',
+    'spark.kubernetes.executor.label.yelp.com/paasta_instance',
```

Comment:
Can you also add cluster?

https://github.com/Yelp/paasta/blob/5d3523b5ec57c9857137a7cb6cecebf9cd0931f7/paasta_tools/setup_kubernetes_cr.py#L265